pyvlx/api/get_local_time.py

```diff
@@ -1,28 +1,34 @@
 """Module for local time firmware version from API."""
+from typing import TYPE_CHECKING
+
 from pyvlx.dataobjects import DtoLocalTime
 
 from .api_event import ApiEvent
-from .frames import FrameGetLocalTimeConfirmation, FrameGetLocalTimeRequest
+from .frames import (
+    FrameBase, FrameGetLocalTimeConfirmation, FrameGetLocalTimeRequest)
+
+if TYPE_CHECKING:
+    from pyvlx import PyVLX
 
 
 class GetLocalTime(ApiEvent):
     """Class for retrieving firmware version from API."""
 
-    def __init__(self, pyvlx):
+    def __init__(self, pyvlx: "PyVLX"):
         """Initialize GetLocalTime class."""
         super().__init__(pyvlx=pyvlx)
         self.success = False
         self.localtime = DtoLocalTime()
 
-    async def handle_frame(self, frame):
+    async def handle_frame(self, frame: FrameBase) -> bool:
         """Handle incoming API frame, return True if this was the expected frame."""
         if not isinstance(frame, FrameGetLocalTimeConfirmation):
             return False
-        self.localtime = frame.localtime
+        self.localtime = frame.time
```

Comment:
Not sure how the former code worked at all given the lack of the named field?

Reply:
It probably never worked ....